packages/xo-server-netbox/src/index.js

```diff
@@ -284,22 +291,27 @@ class Netbox {
       }
     }
 
-    // FIXME: Should we deduplicate cluster names even though it also fails
-    //        when a cluster within another cluster type has the same name?
     const newClusters = []
-    if (clustersToUpdate.length > 0) {
-      log.info(`Updating ${clustersToUpdate.length} clusters`)
-      newClusters.push(...(await this.#request('/virtualization/clusters/', 'PATCH', clustersToUpdate)))
-    }
-    if (clustersToCreate.length > 0) {
-      log.info(`Creating ${clustersToCreate.length} clusters`)
-      newClusters.push(...(await this.#request('/virtualization/clusters/', 'POST', clustersToCreate)))
-    }
+    newClusters.push(...(await this.#request('/virtualization/clusters/', 'PATCH', clustersToUpdate)))
+    newClusters.push(...(await this.#request('/virtualization/clusters/', 'POST', clustersToCreate)))
+
```

Comment:
The order is important. For example, if you have:

Sync 1:
```
{ id: 1, name: 'Foo' }
```
Sync 2:
```
{ id: 1, name: 'Bar' }
{ id: 2, name: 'Foo' }
```
Then during Sync 2, you need to make sure object `1` is updated before object `2` is created, otherwise you'd get a name conflict.

So the order is always `(DELETE >) PATCH > POST`.